packages/flutter_tools/lib/src/android/android_studio.dart

```diff
@@ -83,6 +91,7 @@ class AndroidStudio implements Comparable<AndroidStudio> {
   final String studioAppName;
   final Version version;
   final String configured;
+  final String pathsSelectorPath;
```

Comment:
If it isn't named in a platform specific way, might as well extract lines 109-119 out into the constructors. And add dartdocs for the constructors for targeted platforms.